[add-key-field] add signing key and update readme (#10)
kaleido-io/developer-challenge: README.md

## Setting up your FireFly on your machine

1. Install the [FireFly CLI here](https://github.com/hyperledger/firefly-cli?tab=readme-ov-file#install-the-cli)
2. Create a FireFly stack by running:
   ```bash

>    ff init devChallenge --block-period 2 # --block-period 2 is required for this project

Reviewer: ```suggestion
   ff init devChallenge --block-period 2 # Please set this. We expect you to use 2 second block period for this project (as real world blockchains are not instantaneous)
```

>    ff init devChallenge --block-period 2 # --block-period 2 is required for this project

Reviewer: ```suggestion
   ff init dev
```

We should avoid casing in the name. `dev` seems fine to me, but totally good for you to make it `devchallenge` or `dev_challenge` - but think it will introduce confusion if it's `devChallenge`

   ```
3. Start the FireFly stack by running:
   ```bash

>    ff start devChallenge

Reviewer: ```suggestion
   ff start dev
```

We should avoid casing in the name. `dev` seems fine to me, but totally good for you to make it `devchallenge` or `dev_challenge` - but think it will introduce confusion if it's `devChallenge`

   ```
4. When you're done, you will have FireFly and all its microservices, including your very own private blockchain, running on your machine.

> If you run into issues, use the following resources to help:

Reviewer: ```suggestion
> If you are on Windows or Linux, please **make sure you read the** the hints and tips on [this page](https://hyperledger.github.io/firefly/latest/gettingstarted/firefly_cli/)

If you run into issues, use the following resources to help:
```
